pkg/image/importer/importer_test.go

```diff
@@ -0,0 +1,19 @@
+package importer
+
+import (
+	"net/http"
+	"testing"
+
+	kapi "k8s.io/kubernetes/pkg/api"
+
+	"github.com/openshift/origin/pkg/image/api"
+)
+
+func TestImportNothing(t *testing.T) {
```

Comment:
the integration test imagestreamimporter covers the actual scenarios, but I'll add some unit tests to the core code.